appstore/api/v1/serializers.py

```diff
@@ -23,7 +23,7 @@ class InstanceSerializer(serializers.Serializer):
     aid = serializers.CharField(allow_null=True)
     sid = serializers.CharField()
     fqsid = serializers.CharField()
-    workspace_name = serializers.CharField()
+    workspace_name = serializers.CharField(allow_blank=True)
```

Comment:
@n0mn0m Greatly appreciate your input. Made this change, please let me know if this would suffice.

Reply:
Looks good to me.